components/mpas-ocean/src/shared/mpas_ocn_vmix.F

```diff
@@ -1583,9 +1583,26 @@ subroutine ocn_vmix_implicit(dt, meshPool, statePool, forcingPool, scratchPool,
                !$acc enter data copyin(tracerGroupSurfaceFlux)
 #endif
 
-               call ocn_tracer_vmix_tend_implicit(dt, vertDiffTopOfCell, layerThickness, tracersGroup, &
+               if (trim(groupItr % memberName) /= 'activeTracers' .and. &
+                   config_cvmix_background_diffusion_passive_enable) then
+                      vertDiffPassiveTopOfCell = vertDiffTopOfCell - &
+                         config_cvmix_background_diffusion + config_cvmix_background_diffusion_passive
```

Comment:
For multiple passive tracers this line of code is redundant. It could be placed above the tracer loop, but it is way up there, so for a few wasted flops this more clear.

Reply:
@mark-petersen yes I debated this one in my head a bit, and also decided clarity might might be worth some redundancy.  I'll make a comment in the code but leave the logic as-is if that's OK with you.